sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -749,24 +749,24 @@ def share(
     def mount(
         self,
         path: str,
+        *,
         mount_point: Optional[str] = None,
         mode: str = "ro_mount",
         debug: bool = False,
         persistent: bool = False,
-        **_kwargs,
```

Comment:
Please don't remove kwargs - it must be supported publicly and passed down to the underlying generated layer.
It should just be renamed to remove the underscore prefix.
Same applies to the datastore operation below.